### DISCUSSION TITLE
Unable to get github acccount back

### DISCUSSION THREAD

**Discussion by devbykushal** [community/community#139459] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I had an account with MFA and unable to log back in due to MFA  but i have access to the email! 
I have sent github support request (8 days ago) but still the work is not done, are they they that busy or is it not noticed?

**Comment by ebndev** [comment, marked as answer]
Hi @devbykushal,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
